student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please make sure to pass the parameters in the correct sequence in the super method as expected in the parent class. ` super(age: age, name: name, parent_permission: parent_permission)`.